components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthUtil.java

```diff
@@ -125,4 +125,21 @@ public static AuthenticatedUser getAuthenticatedUser(String fullyQualifiedUserNa
         return authenticatedUser;
     }
 
+    /**
+     * This is used to handle the OAuthAdminService exceptions. This will log the error message and return an
+     * IdentityOAuthAdminException exception
+     * @param message error message
+     * @param exception Exception.
+     * @return
+     */
+    public static IdentityOAuthAdminException handleError(String message, Exception exception) {
```

Comment:
How about changing the name to logAndThrowException() :wink:

Reply:
Actually, this only logs and returns the Exception.